content/_includes/site/cta.njk

```diff
@@ -1,30 +1,25 @@
 {% import 'embed.macros.njk' as embed %}
+{% import 'contact.macros.njk' as contact %}
 
-{% set cta = collections.all | findPage('data.cta_slug', cta or 'default') %}
+{%- set title = 'Want to chat about developing your web app or design system?' -%}
+{%- set contents -%}
+  Contact us for custom web application design & development
+  with our team of Python, UX, and front-end experts,
+  or for an audit to improve your design system.
+  **Fill out the form below
+  or [pick a time](https://calendly.com/oddbird/discovery) for a call.**
+{%- endset -%}
```

Comment:
I added these so we can process them as Markdown, which adds e.g. `<span class="amp">&amp;</span>`.